### PR TITLE
Add Actually Additions handlers

### PR DESCRIPTION
**Pull request by ilkerbey-01**
I have no experience in coding and not sure if this works, but here is my try at adding Actually Additions handlers to NEI

### CODE REVIEW

File: src/main/resources/assets/nei/csv/handlers.csv [hunk 1]
```diff
@@ -511,3 +511,8 @@ tonius.neiintegration.mods.minefactoryreloaded.RecipeHandlerSewer,MineFactory Re
 tonius.neiintegration.mods.minefactoryreloaded.RecipeHandlerGrinder,MineFactory Reloaded,MineFactoryReloaded:machine.0:13,,MineFactoryReloaded,TRUE,,,,,,,,,,,
 tonius.neiintegration.mods.minefactoryreloaded.RecipeHandlerBioReactor,MineFactory Reloaded,MineFactoryReloaded:machine.1:10,,MineFactoryReloaded,TRUE,,,,,,,,,,,
 tonius.neiintegration.mods.minefactoryreloaded.RecipeHandlerSlaughterhouse,MineFactory Reloaded,MineFactoryReloaded:machine.1:13,,MineFactoryReloaded,TRUE,,,,,,,,,,,
+de.ellpeck.actuallyadditions.mod.nei.NEIBookletRecipe,Actually Additions,ActuallyAdditions:itemBooklet,,Actually Additions,TRUE,,,,,,,,,,,
```
Comment (miozune) [inline src/main/resources/assets/nei/csv/handlers.csv:514]: `modId` is `ActuallyAdditions`, so this should be `de.ellpeck.actuallyadditions.mod.nei.NEIBookletRecipe,Actually Additions,ActuallyAdditions:itemBooklet,,ActuallyAdditions,TRUE,,,,,,,,,,,` instead. Same for others.

File: src/main/resources/assets/nei/csv/handlers.csv [hunk 2]
```diff
@@ -511,3 +511,8 @@ tonius.neiintegration.mods.minefactoryreloaded.RecipeHandlerSewer,MineFactory Re
...
+de.ellpeck.actuallyadditions.mod.nei.NEIHairyBallRecipe,Actually Additions,ActuallyAdditions:itemHairyBall,,Actually Additions,TRUE,,,,,,,,,,,
+de.ellpeck.actuallyadditions.mod.nei.NEICrusherRecipe$double,Actually Additions,ActuallyAdditions:blockGrinderDouble,,Actually Additions,TRUE,,,,,,,,,,,
```
Comment (miozune) [inline src/main/resources/assets/nei/csv/handlers.csv:516]: `$Double`

File: src/main/resources/assets/nei/csv/handlers.csv [hunk 3]
```diff
@@ -511,3 +511,8 @@ tonius.neiintegration.mods.minefactoryreloaded.RecipeHandlerSewer,MineFactory Re
...
+de.ellpeck.actuallyadditions.mod.nei.NEICrusherRecipe,Actually Additions,ActuallyAdditions:blockGrinder,,Actually Additions,TRUE,,,,,,,,,,,
+de.ellpeck.actuallyadditions.mod.nei.NEIFurnaceRecipe$double,Actually Additions,ActuallyAdditions:blockFurnaceDouble,,Actually Additions,TRUE,,,,,,,,,,,
```
Comment (miozune) [inline src/main/resources/assets/nei/csv/handlers.csv:518]: `NEIFurnaceDoubleRecipe`

**Review by miozune — CHANGES_REQUESTED** [review]
- Also edit `catalysts.csv` to make machines show up on left side
- Some handlers are missing, see [here](https://github.com/Ellpeck/ActuallyAdditions/tree/1.7.10/src/main/java/de/ellpeck/actuallyadditions/mod/nei)
- You can run `gradlew runClient` or `gradlew build` to see if it's actually working. Feel free to ask on GTNH Discord for help

**Comment by ilkerbey-01** [comment]
should be good now?

**Comment by miozune** [comment]
These are missing:
- NEICoffeeMachineRecipe
- NEICompostRecipe
- NEIReconstructorRecipe
- NEITreasureChestRecipe

**Comment by miozune** [comment]
Swap `ActuallyAdditions:blockCompost` and `ActuallyAdditions:blockAtomicReconstructor`

**Review by ilkerbey-01 — COMMENTED** [review]
that should solve it?

File: src/main/resources/assets/nei/csv/handlers.csv [hunk 4]
```diff
@@ -511,3 +511,13 @@ tonius.neiintegration.mods.minefactoryreloaded.RecipeHandlerSewer,MineFactory Re
...
 tonius.neiintegration.mods.minefactoryreloaded.RecipeHandlerSlaughterhouse,MineFactory Reloaded,MineFactoryReloaded:machine.1:13,,MineFactoryReloaded,TRUE,,,,,,,,,,,
+de.ellpeck.actuallyadditions.mod.nei.NEIBookletRecipe,Actually Additions,ActuallyAdditions:itemBooklet,,ActuallyAdditions,TRUE,,,,,,,,,,,
+de.ellpeck.actuallyadditions.mod.nei.NEIHairyBallRecipe,Actually Additions,ActuallyAdditions:itemHairyBall,,ActuallyAdditions,TRUE,,,,,,,,,,,
+de.ellpeck.actuallyadditions.mod.nei.NEICrusherRecipe$Double,Actually Additions,ActuallyAdditions:blockGrinderDouble,,ActuallyAdditions,TRUE,,,,,,,,,,,
+de.ellpeck.actuallyadditions.mod.nei.NEICrusherRecipe,Actually Additions,ActuallyAdditions:blockGrinder,,ActuallyAdditions,TRUE,,,,,,,,,,,
+de.ellpeck.actuallyadditions.mod.nei.NEIFurnaceDoubleRecipe,Actually Additions,ActuallyAdditions:blockFurnaceDouble,,ActuallyAdditions,TRUE,,,,,,,,,,,
+de.ellpeck.actuallyadditions.mod.nei.NEICoffeeMachineRecipe,Actually Additions,ActuallyAdditions:blockCoffeeMachine,,ActuallyAdditions,TRUE,,,,,,,,,,,
+de.ellpeck.actuallyadditions.mod.nei.NEICompostRecipe,Actually Additions,ActuallyAdditions:blockAtomicReconstructor,,ActuallyAdditions,TRUE,,,,,,,,,,,
```
Comment (miozune) [inline src/main/resources/assets/nei/csv/handlers.csv:520]: and `ActuallyAdditions:blockCompost` here

**Review by ilkerbey-01 — COMMENTED** [review]
now?

**Review by miozune — APPROVED** [review]
All good now 👍

**Pull request merged**